signetdev/host/signetdev_unix.c

```diff
@@ -23,8 +24,10 @@ void signetdev_close_connection()
 
 void signetdev_priv_platform_init()
 {
-	pipe(g_command_pipe);
-	pipe(g_command_resp_pipe);
+	if (pipe(g_command_pipe) == -1 || pipe(g_command_resp_pipe) == -1) {
+		perror("Could not oppen pipe!");
```

Comment:
There is a typo in the print message bug this is a case that could warrant a print but I think there should be some kind of common prefix for all catastrophic failures of this sort. Like "Signet device: Could not open pipe!"

Reply:
Ok, fixed.